docs/tutorials/using-xv11-lidar.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Using the XV11 LIDAR
+subject: Hardware - UART
```

Comment:
I'm thinking maybe we call this `Hardware - Other` since it is more than just UART.